pkg/steps/artifacts.go

```diff
@@ -366,27 +367,6 @@ func addPodUtils(pod *coreapi.Pod, artifactDir string, decorationConfig *prowv1.
 	return nil
 }
 
-func addArtifacts(pod *coreapi.Pod, artifactDir string) {
```

Comment:
This is suspicious but I think templates don't actually use this, they use `addArtifactsToPod` ...